WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -977,7 +979,8 @@ private void trackLastVisiblePage(PageType pageType, boolean trackAnalytics) {
             case READER:
                 ActivityId.trackLastActivity(ActivityId.READER);
                 if (trackAnalytics) {
-                    AnalyticsTracker.track(AnalyticsTracker.Stat.READER_ACCESSED);
+                    AnalyticsTracker.track(AnalyticsTracker.Stat.READER_ACCESSED,
+                            mBiasAAExperiment.getEventProperties());
```

Comment:
Will these extra eventrops only be attached if an experiment variation was successfully assigned? If a user is offline and fails to retrieve a variation assignment, then we will not want to add the extra eventprops.

Reply:
As of right now, these eventprops are always going to be attached to the event. My understanding was that we wanted the API request to be abstracted from the client, meaning it should only care if the variation is `control` or `treatment`. If we want to make the client aware of whether or not the request process finished successfully, then we're effectively checking if the variation is `control`, `treatment` or `unknown`, but we have just removed that from the implementation in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1845 as suggested by @withinboredom.

Sorry for the miscommunication there, I think it's now clear we had different understandings on how we should handle this scenario. I'm going to discuss with @leandroalonso to decide what's the best path from here.

Reply:
I guess the reason here is if the experiment is run only on a subset of users and you want the user groups (and the control) to have the same size, right?

Reply:
We've decoded to drop the eventprop requirement, so I'm going to mark this thread as resolved!